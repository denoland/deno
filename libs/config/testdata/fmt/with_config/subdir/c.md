## Permissions

Deno is secure by default. Therefore,
unless you specifically enable it, a
program run with Deno has no file,
network, or environment access. Access
to security sensitive functionality
requires that permisisons have been
granted to an executing script through
command line flags, or a runtime
permission prompt.

For the following example `mod.ts` has
been granted read-only access to the
file system. It cannot write to the file
system, or perform any other security
sensitive functions.
