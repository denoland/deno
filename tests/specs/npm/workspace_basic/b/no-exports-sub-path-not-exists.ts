import * as c from "@denotest/c/non-existent";
console.log(c);
