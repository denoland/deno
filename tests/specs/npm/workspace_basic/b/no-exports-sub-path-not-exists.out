[# not the best error, but it did resolve because there was no exports specified]
error: [ERR_MODULE_NOT_FOUND] Cannot find module 'file:///[WILDLINE]/c/non-existent' imported from 'file:///[WILDLINE]/b/no-exports-sub-path-not-exists.ts'
    at file:///[WILDLINE]/b/no-exports-sub-path-not-exists.ts:1:20
